packages/n/ncnn/xmake.lua

```diff
@@ -0,0 +1,46 @@
+package("ncnn")
+    set_homepage("https://github.com/Tencent/ncnn")
+    set_description("High-performance neural network inference framework optimized for the mobile platform")
+    set_license("BSD-3-Clause")
+
+    add_urls("https://github.com/Tencent/ncnn.git", {submodules = false})
+    add_versions("20250503", "305837fd4a722ebc47c5d72e72d8ec9ae970e932")
+
+    add_configs("vulkan", {description = "Enable Vulkan support", default = false, type = "boolean"})
+    add_configs("shared", {description = "Build shared library", default = false, type = "boolean"})
+
+    add_deps("cmake")
+    add_deps("protobuf-cpp v3.11.2", "glslang")
+    add_links("ncnn")
+    on_load("windows", function (package)
+        if package:config("vulkan") then
+            package:add("deps", "vulkansdk")
+        end
+    end)
+
+    on_load("linux", "macosx", function (package)
```

Comment:
```suggestion
    on_load("linux", "macosx", "cross", function (package)
```
It looks likes cross can be built if libomp symbols would work.